### PR TITLE
Regenerate `jobs.yaml` and `labels.yaml`

### PR DESCRIPTION
**Pull request by A-Hilaly**
Regenerate `jobs.yaml` and `labels.yaml`

Signed-off-by: Amine Hilaly <hilalyamine@gmail.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prow/jobs/jobs.yaml [hunk 1]
```diff
@@ -3285,7 +3285,7 @@ postsubmits:
           - name: GITHUB_TOKEN
             valueFrom:
               secretKeyRef:
-                name: prowjob-github-token
+                name: prowjob-github-pat-token
```
Comment (A-Hilaly) [inline prow/jobs/jobs.yaml:3288]: Not sure if this is correct, @jljaco @RedbackThomson ?

File: prow/jobs/labels.yaml [hunk 2]
```diff
@@ -525,6 +525,11 @@ repos:
       name: service/emrcontainers
       target: issues
       addedBy: anyone
+    - color: f59745
+      description: Indicates issues or PRs that are related to eventbridge-controller.
+      name: service/eventbridge
+      target: issues
+      addedBy: anyone
```
Comment (A-Hilaly) [inline prow/jobs/labels.yaml:532]: I forgot to regenerate labels.yaml my last PR to test-infra

File: prow/jobs/jobs.yaml (hunk 1 above)
Comment (jljaco) [reply prow/jobs/jobs.yaml:3288]: I believe this is correct

https://github.com/aws-controllers-k8s/test-infra/blob/9b24d142619fd2058415065b7e3ec3a11900b892/infra/lib/charts/prow-secrets.ts#L51

**Review by jljaco — APPROVED** [review]

**Comment by RedbackThomson** [comment]
I genuinely forget to generate labels each time I rebuild the jobs. I now see we have a `gen-all`, but could we also make the default `make` target just call `gen-all` so I don't have to remember?

Otherwise this just needs another rebase and we can merge

**Pull request closed without merging**